website/django/data/models.py

```diff
@@ -9,9 +9,11 @@ class DataRelease(models.Model):
     sources = models.TextField()
     md5sum = models.TextField()
     created = models.DateTimeField(auto_now_add=True, null=True)
+    name = models.CharField(max_length=255)
```

Comment:
I'm wondering if we should set this to some integer type and not a char type. Some parts of this PR depend on the fact that a release name is an integer. I get that the release name might change to something else at some point, however, it seems safer to force another DB migration in case this happens.